integration/helpers.go

```diff
@@ -77,6 +78,218 @@ const (
 	Host     = "localhost"
 )
 
+const (
+	HostID = "00000000-0000-0000-0000-000000000000"
+	Site   = "local-site"
+)
+
+type integrationTestSuite struct {
+	me *user.User
+	// priv/pub pair to avoid re-generating it
+	priv []byte
+	pub  []byte
+	// log defines the test-specific logger
+	log utils.Logger
+}
+
+func newSuite(t *testing.T) *integrationTestSuite {
+	SetTestTimeouts(time.Millisecond * time.Duration(100))
```

Comment:
I think this can be removed. @jakule ?

Reply:
It looks like this was a merge conflict, it's not in `master`.

Reply:
`SetTestTimeouts()` call should definitely be here. If it's not in `master` then it probably explains some of the integration test timeouts we've been seeing.

Reply:
Here's the PR for it from today, though I haven't looked into the details - https://github.com/gravitational/teleport/pull/13312

Reply:
I removed it today as Brian mentioned.